django-nonrel/ocl/collection/models.py

```diff
@@ -68,12 +75,29 @@ def clean(self):
             self.references.append(ref)
             object_version = CollectionVersion.get_head(self.id)
             ref_hash = {'col_reference': ref}
+
             error = CollectionVersion.persist_changes(object_version, **ref_hash)
             if error:
                 errors[expression] = error
+
         if errors:
             raise ValidationError({'references': [errors]})
 
+    def add_expression_concept_related_mappings(self):
```

Comment:
The jargon within the project for this is "direct_mappings" I suggest you rename the method to add_direct_mappings, and include expression as a method parameter